Update README.md (#3)
DevExpress-Examples/HtmlEditor-for-DevExtreme---How-to-implement-mail-merge-with-variables: README.md

<!-- default badges end -->
# HtmlEditor for DevExtreme - How to implement mail merge with variables

The HtmlEditor component supports [variables](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/variables/). This example demonstrates how to implement mail merge with variables.

> The application contains HtmlEditor and two TextBoxes. Type a value in the TextBoxes to see it merged with HtmlEditor text.

Reviewer: `Type a value in the TextBoxes to see it merged with HtmlEditor text.`

1. Is "TextBox" the actual control name? Or are you talking about "text boxes" in a general sense?

2. Are you only typing one value (the same value) in both TextBoxes? Or are you telling the reader to type a separate value in each TextBox:

`Type a value in each TextBox to see it merged with HtmlEditor text.`

Author: These are different values. For example, the first TextBox is for the name insert, the other one is for surname insert.

Reviewer: Okay, since they are different values, can we update this line to the following?

```suggestion
The application contains HtmlEditor and two TextBoxes. Type values in the TextBoxes to see them merged with HtmlEditor text.
```
